app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -208,83 +208,107 @@ class HomeActivityTest {
   }
 
   @Test
-  fun testHomeActivity_withAdminProfile_profileNameIsDisplayed() {
+  fun testHomeActivity_hasCorrectActivityLabel() {
+    launch(HomeActivity::class.java).use { scenario ->
+      scenario.onActivity { activity ->
+        val title = activity.title
+        assertThat(title).isEqualTo(context.getString(R.string.home_activity_title))
+      }
+    }
+  }
+
+  @Test
+  fun testHomeActivity_withMorningTimeAndAdminProfile_MorningWithProfileNameIsDisplayed() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_FIXED_FAKE_TIME)
+    fakeOppiaClock.setCurrentTimeToSameDateTime(MORNING_TIMESTAMP)
     launch<HomeActivity>(createHomeActivityIntent(internalProfileId)).use {
       testCoroutineDispatchers.runCurrent()
       scrollToPosition(position = 0)
       verifyExactTextOnHomeListItemAtPosition(
         itemPosition = 0,
-        targetViewId = R.id.profile_name_text_view,
-        stringToMatch = "Admin!"
+        targetViewId = R.id.welcome_text_view,
+        stringToMatch = "Good morning,Admin!"
       )
     }
   }
 
   @Test
-  fun testHomeActivity_hasCorrectActivityLabel() {
```

Comment:
@yashbansal130 Also why did you remove this `testHomeActivity_hasCorrectActivityLabel` test. Please add it again as it tests activity label and create a separate test for `testHomeActivity_withMorningAndAdminProfile_configChange_MorningWithProfileNameIsDisplayed`.

Reply:
it was removed by mistake, I will restore it.

Reply:
> @yashbansal130 Also why did you remove this `testHomeActivity_hasCorrectActivityLabel` test. Please add it again as it tests activity label and create a separate test for `testHomeActivity_withMorningAndAdminProfile_configChange_MorningWithProfileNameIsDisplayed`.

it is not removed it is is shifted from this line, go to line number 211

Reply:
@yashbansal130 please try not to shift the existing code unless needed as it makes code review a little confusing, and we have to do extra work to figure out the changes.